src/panels/CreateClusterPanel.ts

```diff
@@ -46,6 +47,14 @@ export class CreateClusterDataProvider implements PanelDataProvider<"createClust
         };
     }
 
+    getTelemetryDefinition(): TelemetryDefinition<"createCluster"> {
+        return {
+            getResourceGroupsRequest: false,
+            getLocationsRequest: false,
+            createClusterRequest: true,
```

Comment:
@peterbom I am assuming you are still working on this PR so that we are NOT logging whole createClusterRequest here correct ? we might need just args like preset selected, command was run. nothing else.

Reply:
@peterbom never mind, got it. if it just true, it would log just the command is ran. make sense. Thanks.

Reply:
Absolutely! I realise my original PR description wasn't clear, so I updated that. In short, the `createClusterRequest: true` here means an event will be emitted containing:
```
EventName: "vscode-aks-tools/command"
Properties: {
   command: "createCluster.createClusterRequest"
}
```
There won't be any other data in there.